### DISCUSSION TITLE
Deployment of ITs

### DISCUSSION THREAD

**Discussion by bernt-matthias** [galaxyproject/galaxy#18592] [General Discussion]
ITs are currently bundled with Galaxy's code base. Some are in [EU's fork](https://github.com/usegalaxy-eu/galaxy/tree/release_24.1_europe/tools/interactive). Maybe there are other places that I do not know of :) 

Deployment via TS would be nice regarding deployment and versioning, but there are still concerns in the backend group. 

One option would be to get more of them from EU's fork to the main repo. Another alternative might be a separate repo for ITs.

**Comment by bgruening** [comment]
I consider EUs ITs as a staging area for not yet production-ready ITs. If something is ready it should move upstream. But TS integration would be the best +1.

**Reply by bernt-matthias** [reply]
Also having container definition and creation at a central place would be nice (at least for the tools in upstream).

**Comment by mvdbeek** [comment]
FWIW https://toolshed.g2.bx.psu.edu/view/goeckslab/chatanalysis/eaa8cead8adb is an IT on the tool shed. I kind of think it'd be better to just upload them there for the reasons mentioned. If we consider the IT interface not stable at this point there are other ways to communicate that ? In a way inclusion in the main codebase is not exactly saying "we're not comfortable with this yet"
